include/inexor/vulkan-renderer/world/cube.hpp

```diff
@@ -31,17 +31,76 @@ using Polygon = std::array<glm::vec3, 3>;
 
 using PolygonCache = std::shared_ptr<std::vector<Polygon>>;
 
+/// IDs of the children in a order so that we rotate the octants when moving the child with the first ID to the second ID,
+/// the second, to the third, ...
+using ChildRotationOrder = std::array<std::array<uint8_t, 4>, 2>;
+
+/// IDs of the edges in a order so that we rotate the cube when moving the edge with the first ID to the second ID,
+/// the second, to the third, ...
+/// As we need to update the indentations, it is required that first two indentations in the array
+/// point in the correct order towards the same corner as the second two corners. Example:
+///    -1->       Correct are either {{0, 1, 3, 2}} or {{3, 2, 0, 1}}
+///   ^    ^      Incorrect are all other variants.
+/// 0 |    | 2
+///    -3->
```

Comment:
1. Documentation should be up to date, no matter which developing stage or we gonna end up like the Unity Engine Documentation soon (unusable after a version update)
2. Yes you are right here. My thoughts were a bit off.

About 2. better do this

```suggestion
/// IDs of the edges in a order so that we rotate the cube when moving the edge with the first ID to the second ID,
/// the second, to the third, ...
/// As we need to update the indentations, it is required that first two indentations in the array
/// point in the correct order towards the same corner as the second two corners. Example:
/// Correct are either ``{{0, 1, 3, 2}}`` or ``{{3, 2, 0, 1}}``. Incorrect are all other variants.
///
///        -1->
///       ^    ^
///     0 |    | 2
///        -3->
```
- Added the ` `` ` to get a better visualisation 
- Added 4 spaces and a new line in front of the diagram to make it a code block which has a monospace font